lib/views/admin_request/hidden_user_explanation/_personal_information.text.erb

```diff
@@ -1,3 +1,20 @@
 <%= _('We consider it is not a valid FOI request as it was correspondence ' \
       'about your personal circumstances, and we have therefore hidden it ' \
       'from other users.') %>
+
+<%= _('* If you would like to know what information a public authority holds ' \
+      'about you, you should make a “Subject Access Request” in private ' \
+      'Data Protection law. The Information Commissioner’s website provides '\
+      'advice on accessing your own personal information: ' \
+      'https://ico.org.uk/your-data-matters/your-right-of-access') %>
```

Comment:
[How to access information from a public body](https://ico.org.uk/your-data-matters/official-information/) looks like a more up-to-date URL.

Reply:
The URL in the template is one of the ICO's new GDPR compliant help pages - it's linked from https://ico.org.uk/your-data-matters/ as "Your right to get copies of your data"